### PR TITLE
sl-177: Edit portfolio section functionality

### PR DESCRIPTION
**Pull request by vbazhynov**
image
image
image

### CODE REVIEW

File: frontend/src/packages/sections/libs/validation-schemas/validation-schemas.ts [hunk 1]
```diff
@@ -1 +1,2 @@
 export { siteHeaderUpdateContentValidationSchema } from 'shared/build/index.js';
+export { sitePortfolioUpdateContentValidationSchema } from 'shared/build/index.js';
```
Comment (jaroslawtt) [inline frontend/src/packages/sections/libs/validation-schemas/validation-schemas.ts]:
```suggestion
export { siteHeaderUpdateContentValidationSchema, sitePortfolioUpdateContentValidationSchema } from 'shared/build/index.js';
```

Comment (jaroslawtt) [reply frontend/src/packages/sections/libs/validation-schemas/validation-schemas.ts]: merge?

**Review by what1s1ove — CHANGES_REQUESTED** [review]
waiting for comments fixes and conflicts fixes

File: frontend/src/packages/sections/libs/validation-schemas/validation-schemas.ts (hunk 1 above)
Comment (vbazhynov) [reply frontend/src/packages/sections/libs/validation-schemas/validation-schemas.ts]: thnx

**Review by Izdormu — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**